AirportPanel/Program.cs

```diff
@@ -0,0 +1,490 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AirportPanel
+{
+    enum CityName
+    {
+        Stanbul = 1,
+        Kiev,
+        Athens,
+        Rome,
+        Paris,
+        Berlin,
+        Prague,
+        Warsaw,
+    }
+    enum AirlineName
+    {
+        May = 1,
+        TurkishAirlines,
+        Lufthansa,
+        YanAir,
+        AeroJet,
+    }
+    enum TerminalName
+    {
+        A = 1,
+        B,
+    }
+    enum FlightStatusName
+    {
+        CheckIn,
+        GateClosed,
+        Arrived,
+        Departed,
+        Unknown,
+        Canceled,
+        Expected,
+        Delayed,
+        InFlight
+    }
+    struct Flight // структура информации о рейсе
+    {
+        public bool InOut; // in - true, out - false
+        public DateTime DTFlight;
+        public int Number;
+        public CityName City;
+        public AirlineName Airline;
+        public TerminalName Terminal;
+        public int Gate;
+        public FlightStatusName Status;
+
+        public void DisplayInfo(int left, int top) // построение строки информации о рейсе
+        {
+            string s = null;
+            Console.SetCursorPosition(left, top);
+
+            s += "| " + String.Format("{0:d3}", Number) + " ";
+            s += "| " + String.Format("{0:g}", DTFlight) + " ";
+            s += "| " + String.Format(City + new string(' ', 7 - City.ToString("g").Length)) + " ";
+            s += "| " + String.Format(Airline + new string(' ', 15 - Airline.ToString("g").Length)) + " ";
+            s += "| " + Program.CentrStr(Terminal.ToString("g"), 7);
+            s += "| " + String.Format("{0:d3}", Gate) + " ";
+            s += "| " + String.Format(Status + new string(' ', 10 - Status.ToString("g").Length)) + " ";
+            s += "| ";
+
+            Console.Write(s);
+        }
+    }
+    class Program
```

Comment:
Bad practic write long class. You must remember about [KISS](https://en.wikipedia.org/wiki/KISS_principle), an acronym for "keep it simple, stupid"

Reply:
how to simplify the withdrawal of the flight string? This is a static method and I brought it to a separate ...

Reply:
This solution is also good, but struct Flight better to put in a separate file.